examples/gke-basic-tiller/variables.tf

```diff
@@ -46,13 +46,12 @@ variable "kubectl_config_path" {
 
```

Comment:
We've been adding concrete types for each of the variables. It's a habit we want to get used to, as it helps to differentiate between cases when we want `number` or `bool` instead of `string`.